### PR TITLE
DM: add dm 5.4.0 performance test report

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
DM: add dm 5.4.0 performance test report

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/8215
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dm/dm-benchmark 5.4.0.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: DM 5.4.0 Benchmark Report
+summary: Learn about the performance of 5.3.0.
```
Comment (TomShawn) [inline dm/dm-benchmark 5.4.0.md:3]:
```suggestion
summary: Learn about the performance of 5.4.0.
```

File: dm/dm-benchmark 5.4.0.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+---
+
+# DM 5.4.0 Benchmark Report
+
+The performance of TiDB Data Migration (DM) v5.4.0 is nearly the same as that of TiDB DM v5.3.0. Therefore, you can refer to [DM 5.3.0 Benchmark Report](/dm/dm-benchmark-v5.3.0.md) to learn about the test purpose, environment, scenarios, and results of DM v5.3.0.
```
Comment (TomShawn) [inline dm/dm-benchmark 5.4.0.md:8]:
```suggestion
The benchmark testing data of TiDB Data Migration (DM) v5.4.0 is nearly the same as that of DM v5.3.0. To learn the test purpose, environment, scenarios, and results of DM v5.3.0, refer to [DM 5.3.0 Benchmark Report](/dm/dm-benchmark-v5.3.0.md). 
```

File: dm/dm-benchmark 5.4.0.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (TomShawn) [inline dm/dm-benchmark 5.4.0.md:1]: Please pay attention to the file name. It should be dm-benchmark-v5.4.0.md.

**Comment by TomShawn** [comment]
@shichun-0415 Please fix the CI error.

**Review by GMHDBJD — APPROVED** [review]
LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**